src/legacy/legacy-default-xpring-client.ts

```diff
@@ -244,7 +245,10 @@ class LegacyDefaultXpringClient implements XpringClientDecorator {
       await this.getBalance(address)
       return true
     } catch (e) {
-      return false
+      if ('code' in e && e.code === grpc.status.NOT_FOUND) {
```

Comment:
I believe best practice here might be to use `hasOwnProperty` vs `in`

Reply:
great, I was actually wondering which one was better so I'll change that.

Reply:
So I just pushed that change  - the linter wanted me to write it like this: Object.prototype.hasOwnProperty.call(e, 'code')   instead of just e.hasOwnProperty('code').  Is that ok?

Reply:
Yeah, thats perfect 👍